templates/projects/external_config.html

```diff
@@ -3,12 +3,23 @@
 {% set active_link  = "settings" %}
 {% set section = _('External Configurations') %}
 {% from "_formhelpers.html" import render_form %}
+{}
 
 {% block projectcontent %}
 {% if not forms %}
 <h3>{{_('No external services have been configured')}}</h3>
 {% else %}
 
+<p style="color:#FF0000";>{{_('WARNING: INVALID FILE LOCATION OR PERMISSIONS MAY RESULT IN <b>LOSS OF DATA</b>.')}}</p>
```

Comment:
It may be a good idea to wrap this code in a check for the config value before rendering:

```python
{% if config.EXT_CONFIG_DOCS %}
  <p>{{_('Before starting your project please confirm the following:')}}</p>
  <ul>
    <li>The bucket and/or directory you plan to access already exists ({{brand}} will not create this for you)</li>
    <li>You have permission to access the directories and/or buckets you plan to use</li>
    <li>You have spelled the bucket and/or directory name correctly</li>
  </ul>

  <p>For more information on these settings, please consult the <a href={{config.EXT_CONFIG_DOCS}}>External Configurations documentation</a>.</p>
{% endif %}
```

Reply:
You can also test this locally by including `EXT_CONFIG_DOCS` in your settings.local and make sure it renders properly. Then remove `EXT_CONFIG_DOCS` and verify the page still renders correctly, but without the extra sentence containing the link. Thanks.

Reply:
Good idea - thanks!